packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Examples.scss

```diff
@@ -1,21 +0,0 @@
-:global {
-  .ms-FacepileExample {
-    max-width: 300px;
-
-    .control {
-      padding-top: 20px;
-    }
-    .ms-Dropdown-container,
```

Comment:
I don't see any equivalent for the dropdown, slider, or checkbox styles in the new styling. (In this case you should use the `styles` prop for all of that.)